ckanext/data_qld/reporting/helpers/helpers.py

```diff
@@ -0,0 +1,222 @@
+import ckan.model as model
+import ckan.plugins.toolkit as toolkit
+import logging
+import pytz
+from datetime import datetime, timedelta
+
+get_action = toolkit.get_action
+log = logging.getLogger(__name__)
+
+
+def check_org_access(org_id):
+    context = get_context()
+    data_dict = {'org_id': org_id, 'permission': 'create_dataset'}
+    if not toolkit.check_access('has_user_permission_for_org', context, data_dict):
+        toolkit.abort(403, toolkit._('User {0} is not authorized to create datasets for organisation {1} test'. format(get_username(), org_id)))
+
+
+def get_context():
+    return {
+        'model': model,
+        'session': model.Session,
+        'user': toolkit.c.user,
+        'auth_user_obj': toolkit.c.userobj
+    }
+
+
+def get_user():
+    return toolkit.c.userobj
+
+
+def get_username():
+    return get_user().name
+
+
+def get_report_date_range(request):
+    start_date = request.GET.get('start_date', None)
+    end_date = request.GET.get('end_date', None)
+
+    if start_date:
+        start_date = toolkit.h.date_str_to_datetime(start_date)
+    else:
+        start_date = datetime(2019, 7, 10)
```

Comment:
Where is this hardcoded value coming from?

Reply:
I'd say that was when salsas work went live, no point looking older.

Reply:
that's correct @duttonw . That date comes from the ACs of the reporting stories. Maybe @ThrawnCA is getting at why it isn't configurable - in which case we can make it configurable.